test/e2e_node/summary_test.go

```diff
@@ -53,9 +55,10 @@ var _ = framework.KubeDescribe("Summary API", func() {
 			const pod1 = "stats-busybox-1"
 
 			By("Creating test pods")
-			createSummaryTestPods(f, pod0, pod1)
-			// Wait for cAdvisor to collect 2 stats points
-			time.Sleep(15 * time.Second)
+			pods := getSummaryTestPods(f, pod0, pod1)
```

Comment:
Can you point to me where we ensure there is no duplicate stats for a single container within your test pod?